<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseWidget</class>
 <widget class="QWidget" name="ChooseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="btn_newTraining">
     <property name="text">
      <string>Training of Neural Net</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_newValidation">
     <property name="text">
      <string>Validation of Neural Net</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_newConversion">
     <property name="text">
      <string>Convert Files</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
